client/app/models/milestone.js

```diff
@@ -107,4 +129,88 @@ export default class MilestoneModel extends Model {
     // get date diff by accounting for DST
     return getTimeDiffText(remainingTimeInterval);
   }
+
+  @computed('project.dcpPublicstatus', 'dcpMilestone', 'dcpActualstartdate', 'dcpActualenddate', 'dcpReviewmeetingdate')
+  get displayDate() {
+    if ([
+      BOROUGH_BOARD_REFERRAL,
+      BOROUGH_PRESIDENT_REFERRAL,
+      COMMUNITY_BOARD_REFERRAL,
+      CITY_COUNCIL_REVIEW,
+      CPC_PUBLIC_MEETING_PUBLIC_HEARING,
+      EIS_DRAFT_SCOPE_REVIEW,
+      FEIS_SUBMITTED_AND_REVIEW,
+      MAYORAL_VOTE,
+    ].includes(this.dcpMilestone) && this.project.get('dcpPublicstatus') !== 717170000) {
```

Comment:
Do you know if we can use native getter syntax for this? `this.project.dcpPublicstatus`. I know that it being a model relation complicates matters for asynchronous relationships.

Reply:
Can `717170000` be a named constant somewhere? Even the top of the file?

Reply:
I would even just define `dcpPublicstatus` at the top `const dcpPublicstatus = ...` just to reduce the cognitive overhead of the syntax (native vs classic getters)

Reply:
Yeah Ember was complaining if I used the native getter syntax -- the error said it needs the Ember getter because of the property is relationship proxy. 

Yes good idea about the constant.  Will change.

Reply:
Ahh okay, I'll get it at the top.